src/GNDStk/misc/src/child.hpp

```diff
@@ -0,0 +1,345 @@
+
+namespace child {
+
+// -----------------------------------------------------------------------------
+// Allowable top-level nodes, per LLNL-TR-774621-DRAFT
+// -----------------------------------------------------------------------------
+
+// Note: the trues mean "allowed as a top-level node"
+inline const child_t<void,allow::one>
+   reactionSuite      ("reactionSuite",       true),
+   covarianceSuite    ("covarianceSuite",     true),
+   PoPs               ("PoPs",                true),
+   thermalScattering  ("thermalScattering",   true),
+   fissionFragmentData("fissionFragmentData", true);
+
+
+
+// -----------------------------------------------------------------------------
+// Keywords
+// Of type child_t
+// -----------------------------------------------------------------------------
+
+// ------------------------
+// one + many pair
+// constructs
+// ------------------------
+
+GNDSTK_MAKE_CHILD(void, averageEnergies, allow::one);
```

Comment:
This file looks like something that could be automated using the JSON files that describe the GNDS format.

Reply:
It may eventually be just that. Or, I may keep these objects in their current form, and have other namespaces to contain objects for, say, version-specific GNDS contents. Lots of ways to handle this sort of stuff.